Misanthropic-Mistletoes/reactive-nutshell
# Sp add new message

# Description

Finished an Add New Message feature.

Fixes https://github.com/Misanthropic-Mistletoes/reactive-nutshell/issues/12

## Type of change

Please delete options that are not relevant.

# Testing Instructions
1) Save and commit your changes to your branch
2) `git checkout master`
3) `git fetch --all`
4) `git checkout sp-add-new-message`
5) Check to see that you can add a new message and the DOM repopulates the Message List INCLUDING the new message.



# Checklist:
